routes/users.js

```diff
@@ -53,6 +103,56 @@ router.post('/', authenticate, userValidator.createUser, usersController.addNewU
  *               $ref: '#/components/schemas/errors/serverUnavailable'
  */
 router.patch('/self', authenticate, userValidator.updateUser, usersController.updateSelf)
+
+/**
+ * @swagger
+ * /users:
+ *   get:
+ *     summary: Get all the users in system.
+ *
+ *     tags:
+ *       - Users
+ *     security:
+ *       - bearerAuth: []
+ *     responses:
+ *       200:
+ *         description: User details
+ *         content:
+ *           application/json:
+ *              schema:
+ *                type: object
+ *                properties:
+ *                  message:
+ *                    type: string
+ *                    example: Users returned successfully!
+ *                  users:
+ *                    type: array
```

Comment:
This array of `$ref` is not working properly, please help.

Reply:
never mind, its done 😄

Reply:
Great! @whyDontI 
For future reference! Thank you.
![image](https://user-images.githubusercontent.com/66248039/98547096-a6f26980-22bd-11eb-9545-7cb59cba57c8.png)